tools/depositCollectionPubs.js

```diff
@@ -19,14 +18,36 @@ const main = async () => {
 	const collection = await Collection.findOne({
 		where: { communityId: community.id, slug: collectionSlug },
 	});
-	const collectionPubs = await CollectionPub.findAll({ where: { collectionId: collection.id } });
-	console.log(`=== Will deposit 1 Collection and ${collectionPubs.length} Pubs`);
+	const collectionId = collection.id;
+	const collectionPubs = await CollectionPub.findAll({
+		where: { collectionId },
+		include: [
+			{
+				model: Pub,
+				as: 'pub',
+				include: [
+					{
+						model: Release,
+						as: 'releases',
+						attributes: ['id'],
+					},
+				],
+			},
+		],
+	});
+	const releasedPubs = collectionPubs.reduce((acc, cp) => {
+		if (cp.pub.releases.length > 0) {
+			acc.push(cp);
+		}
+		return acc;
+	}, []);
```

Comment:
A matter of preference but this could be simplified a little with `filter` e.g.:
```ts
const releasedPubs = collectionPubs.filter(cp => cp.pub.releases.length > 0);
```

Reply:
Thanks! I always forget about filter. I always search map and reduce and forget filter.